redash/query_runner/mongodb.py

```diff
@@ -174,6 +174,13 @@ def _merge_property_names(self, columns, document):
               if property not in columns:
                   columns.append(property)
 
+    def _is_collection_a_view(self, db, collection_name):
+        try:
+            db.command('collstats', collection_name)
+            return True
```

Comment:
When collstats succeeded doesn't it mean this is a Collection rather than a View?

Reply:
Your absolutely right, forgot to switch them before committing